packages/sit-onyx/src/components/OnyxTimepicker/OnyxTimeInput.vue

```diff
@@ -0,0 +1,394 @@
+              v-if="!open || !modelValue"
+              :label="t('timepicker.labels.iconButton')"
+              :icon="iconClock"
+              :color="open ? 'primary' : 'neutral'"
+              class="onyx-timepicker__time-icon"
+              @click.stop="handleOpen"
+            />
+            <button
+              v-else
+              class="onyx-timepicker__clear-button"
+              type="button"
+              :aria-label="t('input.clear')"
+              :title="t('input.clear')"
+              tabindex="-1"
+              @mousedown.prevent
+              @click="modelValue = undefined"
+            >
+              <OnyxIcon :icon="iconXSmall" color="neutral" />
+            </button>
+          </template>
+        </OnyxTimepickerInput>
+      </template>
+
+      <template #content>
+        <div class="onyx-timepicker__wrapper" tabindex="-1">
+          <div class="onyx-timepicker__group">
+            <OnyxStepper
+              ref="hourInputTemplateRef"
+              v-model="hour"
+              :label="t('timepicker.labels.hour')"
+              :placeholder="t('timepicker.placeholder.hour')"
+              :disabled="props.disabled"
+              :loading="props.loading"
+              hide-label
+              hide-clear-icon
+              hide-success-icon
+              hide-buttons
+              @click="handleSegmentFocus(hourInputRef)"
+              @keydown="(e: KeyboardEvent) => handleInputChange('hour', e)"
+            />
+
+            <span class="onyx-timepicker__divider">:</span>
+
+            <OnyxStepper
+              ref="minuteInputTemplateRef"
+              v-model="minute"
+              :label="t('timepicker.labels.minute')"
+              :placeholder="t('timepicker.placeholder.minute')"
+              :disabled="props.disabled"
+              :loading="props.loading"
+              hide-label
+              hide-clear-icon
+              hide-success-icon
+              hide-buttons
+              @click="handleSegmentFocus(minuteInputRef)"
+              @keydown="(e: KeyboardEvent) => handleInputChange('minute', e)"
+            />
+
+            <span v-if="props.showSeconds" class="onyx-timepicker__divider">:</span>
+
+            <template v-if="props.showSeconds">
+              <OnyxStepper
+                ref="secondInputTemplateRef"
+                v-model="second"
+                :label="t('timepicker.labels.second')"
+                :placeholder="t('timepicker.placeholder.second')"
+                :disabled="props.disabled"
+                :loading="props.loading"
+                hide-label
+                hide-clear-icon
+                hide-success-icon
+                hide-buttons
+                @click="handleSegmentFocus(secondInputRef)"
+                @keydown="(e: KeyboardEvent) => handleInputChange('second', e)"
+              />
+            </template>
+          </div>
+
+          <div v-if="props.infoLabel" class="onyx-timepicker__info-label">
```

Comment:
The info label looks pretty huge, is this correct or should it be smaller?

<img width="279" height="189" alt="Image" src="https://github.com/user-attachments/assets/23205af2-7118-4da6-83b8-cf68f51ca103" />